### PR TITLE
formatters for response examples

### PR DESCRIPTION
**Pull request by ZurgInq**
Hello. And sorry, here I will use google translate. I use swagger-ui and it does not display responses examples if there is no scheme or example are not string.
I added the ability to use an arbitrary formatter for examples.
```ruby
RSpec::Rails::Swagger::ResponseFormatters.register('application/json', Proc.new do |resp|
  begin
    resp = JSON.parse(resp)
    resp = JSON.pretty_generate(resp)
  rescue ::JSON::ParserError
    resp
  end
end)
```

### CODE REVIEW

File: lib/rspec/rails/swagger.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 require 'rspec/rails/swagger/configuration'
 require 'rspec/rails/swagger/document'
 require 'rspec/rails/swagger/formatter'
+require 'rspec/rails/swagger/response_formatters'
```
Comment (drewish) [inline lib/rspec/rails/swagger.rb]: Could you move this down below helpers so these are in sorted order?

File: lib/rspec/rails/swagger/response_formatters.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            else
+              resp
+            end
+          rescue ::JSON::ParserError
+            resp
+          end
+        end
+
+        @formatters = {
+            :none              => ->(resp) { resp },
```
Comment (drewish) [inline lib/rspec/rails/swagger/response_formatters.rb]: What do you think about using `:default` for this key?

Comment (drewish) [inline lib/rspec/rails/swagger/response_formatters.rb]: Also could you please use 2 spaces for intending?

**Comment by ZurgInq** [comment]
I made corrections. Please review again. Should I make the squash commit?

**Pull request merged**